es-app/src/views/gamelist/GridGameListView.cpp

```diff
@@ -49,11 +151,245 @@ void GridGameListView::launch(FileData* game)
+		if (gameIter != siblings.end())
+		{
+			if ((gamePos + 1) < siblings.size())
+				setCursor(siblings.at(gamePos + 1));
+			else if ((gamePos - 1) > 0)
+				setCursor(siblings.at(gamePos - 1));
+		}
+	}
+	delete game;                                 // remove before repopulating (removes from parent)
+	onFileChanged(game, FILE_REMOVED);           // update the view, with game removed
+}
+
+void GridGameListView::onThemeChanged(const std::shared_ptr<ThemeData>& theme)
+{
+	ISimpleGameListView::onThemeChanged(theme);
+
+	using namespace ThemeFlags;
+	mGameTitle.applyTheme(theme, getName(), "md_title", ALL);
+	mGrid.applyTheme(theme, getName(), "md_grid", ALL);
+
+	// ---  DETAILED METADATA THEME ---
+	mImage.applyTheme(theme, getName(), "md_image", POSITION | ThemeFlags::SIZE | Z_INDEX | ROTATION);
+
+	initMDLabels();
+	std::vector<TextComponent*> labels = getMDLabels();
+	assert(labels.size() == 8);
+	const char* lblElements[8] = {
+		"md_lbl_rating", "md_lbl_releasedate", "md_lbl_developer", "md_lbl_publisher",
+		"md_lbl_genre", "md_lbl_players", "md_lbl_lastplayed", "md_lbl_playcount"
+	};
+
+	for (unsigned int i = 0; i < labels.size(); i++)
+		labels[i]->applyTheme(theme, getName(), lblElements[i], ALL);
+
+	initMDValues();
+	std::vector<GuiComponent*> values = getMDValues();
+	assert(values.size() == 8);
+	const char* valElements[8] = {
+		"md_rating", "md_releasedate", "md_developer", "md_publisher",
+		"md_genre", "md_players", "md_lastplayed", "md_playcount"
+	};
+
+	for (unsigned int i = 0; i < values.size(); i++)
+		values[i]->applyTheme(theme, getName(), valElements[i], ALL ^ ThemeFlags::TEXT);
+
+	mDescContainer.applyTheme(theme, getName(), "md_description", POSITION | ThemeFlags::SIZE | Z_INDEX);
+	mDescription.setSize(mDescContainer.getSize().x(), 0);
+	mDescription.applyTheme(theme, getName(), "md_description", ALL ^ (POSITION | ThemeFlags::SIZE | ThemeFlags::ORIGIN | TEXT | ROTATION));
+
+	sortChildren();
+}
+
+void GridGameListView::updateInfoPanel()
+{
+	FileData* file = (mGrid.size() == 0 || mGrid.isScrolling()) ? NULL : mGrid.getSelectedObject();
+
+	bool fadingOut;
+	if (file == NULL)
+	{
+		//mImage.setImage("");
+		//mDescription.setText("");
+		fadingOut = true;
+	}
+	else
+    {
+		mGameTitle.setText(file->getName());
+
+		mImage.setImage(file->metadata.get("image"));
+		mDescription.setText(file->metadata.get("desc"));
+		mDescContainer.reset();
+
+        mRating.setValue(file->metadata.get("rating"));
+        mReleaseDate.setValue(file->metadata.get("releasedate"));
+        mDeveloper.setValue(file->metadata.get("developer"));
+        mPublisher.setValue(file->metadata.get("publisher"));
+        mGenre.setValue(file->metadata.get("genre"));
+        mPlayers.setValue(file->metadata.get("players"));
+
+        if (file->getType() == GAME)
+		{
+			mLastPlayed.setValue(file->metadata.get("lastplayed"));
+			mPlayCount.setValue(file->metadata.get("playcount"));
+		}
+
+		fadingOut = false;
+	}
```

Comment:
some indentation issues above

Reply:
Wow thanks for noticing this, my IDE completely fucked up the indentation here for whatever reason it used 4 spaces instead of tabs.

Reply:
This issue is fixed but this comment isn't flagged as "outdated" because you put it on the lines below the problem and not directly on it.